GithubRunnerRegistration/GithubRunnerRegistration/Models/RSAParametersSerializable.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace GithubRunnerRegistration.Models
+{
+    [Serializable]
+    public class RSAParametersSerializable : ISerializable

```

Comment:
This feels like something that we should not be doing by hand and re-use an existing library. Why are we parsing individual RSA parameters? Is this the best library interface available?

Reply:
Your guess is as good as mine https://github.com/actions/runner/blob/be9632302ceef50bfb36ea998cea9c94c75e5d4d/src/Runner.Listener/Configuration/IRSAKeyManager.cs